Add a definition of what is a module compatible multistore (#463)
PrestaShop/docs: src/content/1.7/modules/creation/_index.md

### Handling the Multistore feature

> What is a module compatible multistore ?

Reviewer: ```suggestion
What is a module compatible with the multi-store feature?
```
You can't use one time `multistore` and next time `multi-store`

> A multi-store compatible module is a module that can be configured differently from one store to another.

Reviewer: ```suggestion
It is a module that can be configured differently from one store to another.
```

> It is also a module that can be configured quickly in the same way on all stores thanks to the all shops context or the group of shops.

Reviewer: ```suggestion
Also the same module can have a shared configuration between all shops.
```
wdyt?
